ui/app/AppLayouts/Wallet/services/dapps/WalletConnectService.qml

```diff
@@ -233,4 +245,16 @@ QObject {
         sdk: root.wcSDK
         store: root.store
     }
+
+    Timer {
+        id: timeoutTimer
+
+        interval: 5000 // (5 seconds)
```

Comment:
Is 5 seconds too short, especially on slower network connection, maybe 30 (default connection timeout for many services) would be more suitable?

Reply:
I feel the 30 seconds too much for a websocket low bandwith API involved here. The failing will just prompt user to try again whenever he feels like. Therefore I will increase it to 10 seconds. What do you think?

Reply:
Also this is mainly for the case of not having a response from the SDK in a corner case